package.json

```diff
@@ -6,17 +6,16 @@
   "dependencies": {
     "express": "^4.14.0"
   },
+  "peerDependencies" : {
+    "bs-platform": "file:../bucklescript.git/"
+  },
   "devDependencies": {
-    "bs-platform": "^1.0.1"
+    "bs-platform": "file:../bucklescript.git/"
```

Comment:
@chenglou so devDep + peerDep is the convention right ? That would make sense to me.